accelerator/apis.c

```diff
@@ -9,28 +9,35 @@
 #include "apis.h"
 #include "map/mode.h"
 
+static logger_id_t logger_id;
+
 status_t api_get_tips(const iota_client_service_t* const service, char** json_result) {
   status_t ret = SC_OK;
   get_tips_res_t* res = get_tips_res_new();
   char_buffer_t* res_buff = char_buffer_new();
 
   if (res == NULL || res_buff == NULL) {
     ret = SC_CCLIENT_OOM;
+    log_error(logger_id, "[%s:%d] SC_CCLIENT_OOM\n", __func__, __LINE__);
```

Comment:
It's possible to write status code in `__LINE__`. But I think this should up to you to decide.

Reply:
I think a short sentence to describe the error would be better

Reply:
I think `__LINE___` is better. It would be super helpful for debugging